### PR TITLE
MM-28702 - call InitEnterprise() to initialize Enterprise for cli

### PR DESCRIPTION
**Pull request by sbishel**
#### Summary
Adds Call to InitServer() so that Enterprise Interfaces get initialized for CLI.

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28702

### CODE REVIEW

File: cmd/mattermost/commands/init.go [hunk 1]
```diff
@@ -45,6 +45,7 @@ func InitDBCommandContext(configDSN string) (*app.App, error) {
 
 	if model.BuildEnterpriseReady == "true" {
 		a.Srv().LoadLicense()
+		a.InitEnterprise()
```
Comment (jespino) [inline cmd/mattermost/commands/init.go:48]: Why not just execute here `a.InitServer()`, actually, I think this should be call outside the `BuildEnterpriseReady` if block.

**Review by jespino — APPROVED** [review]
LGTM

**Review by hahmadia — APPROVED** [review]

**Review by srkgupta — APPROVED** [review]
Tested and the issue is working fine now. Approving the PR.

**Comment by amyblais** [comment]
@sbishel Is the release note in the PR description correct?

**Comment by sbishel** [comment]
> Is the release note in the PR description correct?
No, updated...thank you.
cc: @amyblais

**Pull request merged**